Decide/src/main/java/DD2480/decide/Decide.java

```diff
@@ -42,5 +42,16 @@ public Decide( Parameter parameters, Lcm LCM, boolean[] PUV,  Points[] dataPoint
 		return null;
 	}
 
+    private boolean launch(){
```

Comment:
Shouldn't you also add the launch method to Decide() ?

Reply:
Don't think so, launch method is to check if the missile should be fired or not. If I'm not mistaken